File: Player.java
import java.util.Vector;

class Player {
    private Location 	currentLocation;
    private Inventory	inventory = new Inventory();

    public Location getLocation()
    {
        return currentLocation;
    }

    public void setLocation( Location location )
    {
        currentLocation = location;
    }

    public void addItem( Item item )
    {
        inventory.addItem( item );
    }

    public void removeItem( Item item )
    {
        inventory.removeItem( item );
    }

    public Vector getItems()
    {
        return inventory.getItems();
    }
};

File: javascript-conversion/js/Player.js
class Player {
    constructor() {
        this.currentLocation = null;
        this.inventory = new Inventory();
    }

    getLocation() {
        return this.currentLocation;
    }

    setLocation(location) {
        this.currentLocation = location;
    }

    addItem(item) {
        this.inventory.addItem(item);
    }

    removeItem(item) {
        this.inventory.removeItem(item);
    }

    getItems() {
        return this.inventory.getItems();
    }
}